backend/fms_core/template_importer/row_handlers/library_conversion/library_conversion.py

```diff
@@ -0,0 +1,67 @@
+from fms_core.template_importer.row_handlers._generic import GenericRowHandler
+
+from fms_core.services.sample import get_sample_from_container
+from fms_core.services.container import get_container, get_or_create_container
+from fms_core.services.library import convert_library
+
+
+
+class LibraryRowHandler(GenericRowHandler):
+    def __init__(self):
+        super().__init__()
+
+    def process_row_inner(self, library_batch_info, library_source, volume_used, comment,
+                          container, concentration, library_size, volume):
+
+        if not library_batch_info:
+            self.errors['library_conversion'] = 'No batch is associated with this library.'
```

Comment:
you can bail out right there ... It will explode right and left if there is no process.